unitTests/ui/ResultsPerPageTest.ts

```diff
@@ -213,6 +213,19 @@ export function ResultsPerPageTest() {
         });
         expect(test.env.queryController.options.resultsPerPage).toBe(firstChoice);
       });
+
+      it('initialChoice set to non default value is use when default value is in choicesDisplayed', () => {
```

Comment:
I think the description could be clearer. It should mention that the initialChoice value is set to the configured value when the QSM value is the default value. e.g.

```
it (`when the QSM numberOfResults is equal to the default QSM number of results,
when the initialChoice option is not equal to the default QSM numberOfResults,
it sets the resultsPerPage to the initialChoice`, () => { ... })
```

Reply:
just to be sure but QSM is the QueryStateModel?

Reply:
Yep,

Reply:
ok thx